doc-content/enterprise-only/admin-and-config/managing-business-central-user-management-con.adoc

```diff
@@ -0,0 +1,18 @@
+[id='managing-business-central-user-management-con']
+= User management
+
+{CENTRAL} defines three types of entities for security management - users, groups, and roles. A role defines a set of permisions a user or group has in {CENTRAL}. {CENTRAL} provides the following roles:
+
+* process-admin
+* manager
+* admin
+* analyst
+* developer
+* user
+
+[NOTE]
+====
+If a role's identifier is present in the application's Role Registry only then it is considered a role otherwise its a group.
+====
+
+You can use {CENTRAL} to create and manage as many users and groups as you require. A user must be assigned at least one user-specific role to login to {CENTRAL}. User privileges depend on the groups the user is a member of and the roles assigned to the groups.
```

Comment:
"User privileges depend on ..." => "User privileges depends on the permissions collected from the groups and the roles the user is member of. Notice, the role/group priority is taken into account in case the users has several roles/groups assigned."